python/pyarrow/tensor.pxi

```diff
@@ -179,6 +179,29 @@ shape: {0.shape}""".format(self)
                                                c_dim_names, &csparse_tensor))
         return pyarrow_wrap_sparse_coo_tensor(csparse_tensor)
 
+    @staticmethod
+    def from_scipy(obj, dim_names=None):
+        """
+        Convert scipy.sparse.coo_matrix to arrow::SparseCOOTensor
+        """
+        cdef shared_ptr[CSparseCOOTensor] csparse_tensor
+        cdef vector[int64_t] c_shape
+        cdef vector[c_string] c_dim_names
+
```

Comment:
Could check here that the passed object is actually a scipy.sparse.coo_matrix ? 
(right now the user will probably get an attribute error if you pass a different scipy matrix type)

(and the same for the other from_scipy below)

Reply:
Done.

Reply:
Thanks!

Reply:
Thank you :)